api/controllers/users.py

```diff
@@ -34,6 +35,19 @@ def put(self, user_email):
             'user_email': user_id
         }
 
+    @api.doc('delete_user')
+    @api.expect(delete_users_model)
```

Comment:
I dont think this method needs a model? its getting the user_email from the url route so nothing should be in the body of the POST unless we want more information

Reply:
Yeah you are right. The information needed is in the URL so I removed that.